rpm/cray/csm/sle-15sp3/index.yaml

```diff
@@ -27,3 +27,10 @@ https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp3/:
     - craycli-0.71.0-1.x86_64
     - ilorest-3.5.1-1.x86_64
     - libcsm-0.0.4-1.noarch
+
+# To support SP3 storage nodes with SP4 K8S nodes:
+https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp4/:
```

Comment:
This is ridiculous.

Looks like it's time to consider *cough cough* https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5317